### ISSUE TITLE
whitelisting folder (and subfolders)

### ISSUE THREAD

**Issue by XaaRii** [pylorak/TinyWall#17]
Hi! First of all, this project is amazing, good job guys.
I started to use this not too long ago, and it's honestly really solid piece of software. There may be some things that I'd like being differently done, but these are negligible.
There is one feature that I'd like to see being implemented though, and that is an option to unblock executables in a folder (and its subfolders).
To give an example why it would be useful: applications like Discord stores their executable in a folder containing their version (for me it's currently called "app-1.0.72"). When an update happens, that update is downloaded into a new folder and is then used instead. That means the old rules you may had for the previous version are no longer applied. And that's just one example.

I'd love to see something like this being implemented (if this project is still being worked on, that is...).

**Comment by oep42** [comment]
This is a diplicate of https://github.com/pylorak/TinyWall/issues/4

**Comment by ChrisRosser** [comment]
For anyone still subscribed here: I've proposed a fix for the Discord case in
https://github.com/pylorak/TinyWall/pull/133. It doesn't add folder whitelisting — instead, when
an exception's path stops existing, it follows the executable into the sibling version folder
(`app-1.0.72` → `app-1.0.73`) and rewrites the stored path, so rules keep matching one exact
executable.

Details and the caveats on the unofficial build I published are in
https://github.com/pylorak/TinyWall/issues/4#issuecomment-5525886082. Not reviewed by @pylorak
yet.
